mintcoin-qt.pro

```diff
@@ -123,9 +123,12 @@ contains(USE_BUILD_INFO, 1) {
     DEFINES += HAVE_BUILD_INFO
 }
 
-*-g++-32 {
-    message("32 platform, adding -msse2 flag")
-
+!equals($$QMAKE_HOST.arch, armv7l) {
+    message(FOUND host = $$QMAKE_HOST.arch)
+    QMAKE_CXXFLAGS += -mthumb -DNOSSE
+    QMAKE_CFLAGS += -mthumb -DNOSSE
```

Comment:
I did some research into the `-mthumb` flag. This selects the ARM Thumb instruction set, which is a smaller instruction set designed for configurations with a 16-bit memory system:

http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0210c/CACBCAAE.html

This is not really a configuration that makes sense for us, since we currently need more than 1 gibibyte of memory (closer to 2 gibibyte).

However, while the Thumb instruction set usually means more instructions more memory I/O due to fewer registers, this may result in faster code because of the smaller size of instructions. It really depends on the specifics of the program being run.

So I will merge this pull request (PR), but I have opened a separate issue to research the CPU instruction set specifically:

https://github.com/shane-kerr/Mintcoin-Desktop-Wallet/issues/17